apps/aehttp/test/aehttp_sc_SUITE.erl

```diff
@@ -2266,7 +2268,11 @@ sc_ws_dont_open_ga(Config) ->
     RAmt = 4,
 
     ChannelOpts = channel_options(IPubkey, RPubkey, IAmt, RAmt, #{}, Config),
-    {error, rejected} = channel_ws_start(initiator, maps:put(host, <<"localhost">>, ChannelOpts), Config),
+    {ok, IPid} = channel_ws_start(initiator, maps:put(host, <<"localhost">>, ChannelOpts), Config),
+    %% although connection is established, it dies instnatly as the
+    %% participant is a GA
+    timer:sleep(100),
+    undefined = process_info(IPid),
```

Comment:
Shall this rather be a wait-until with larger timeout so to prevent this to fail intermittently?

Though looks good enough.